assets/queries/ansible/aws/sql_analysis_services_port_2383_is_publicly_accessible/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "sql_analysis_services_port_2383_is_publicly_accessible",
+  "queryName": "SQL Analysis Services Port 2383 (TCP) is Publicly Accessible",
+  "severity": "MEDIUM",
+  "category": "Network Ports Security",
+  "descriptionText": "Check if port 2383 on TCP is publicly accessible by checking the CIDR block range that can access it.",
```

Comment:
You are not checking 'CIDR block range' since it doesn't exist in 'amazon.aws.ec2_group' module.